src/staticanalysis/frontend/.neutrinorc.js

```diff
@@ -0,0 +1,48 @@
+fs = require('fs');
+
+const envs = {
+  CONFIG: process.env.CONFIG || 'staging',
+};
+const PORT = process.env.PORT || 8010;
+// HTTPS can be disabled by setting HTTPS_DISABLED environment variable to
+// true. Otherwise it will enforced either using automatically generated
+// certificates or pre-generated ones.
+const HTTPS = process.env.HTTPS_DISABLED ? false :
+  (process.env.SSL_CERT && process.env.SSL_KEY && process.env.SSL_CACERT) ?
```

Comment:
I think we don't set `SSL_CERT`, `SSL_KEY` or `SSL_CACERT`. and new version of `webpack-dev-server` does not include ssl certs anymore. I would like that we refer to our generated ssl certs. Can I push the fix to this branch?

Reply:
I just added you as contributor on my repo, you should be able to push.

This code is also used in `shipit-frontend`.